modules/nodes-pods-autoscaling-about.adoc

```diff
@@ -91,7 +91,7 @@ spec:
   maxReplicas: 7
   minReplicas: 3
   scaleTargetRef:
-    apiVersion: apps.openshift.io/v1
+    apiVersion: apps/v1
```

Comment:
This change should be dropped; in 4.8+ the next line is `kind: Deployment`, unlike here.

Reply:
Thanks. Restored previous version here:
```
  scaleTargetRef:
    apiVersion: apps.openshift.io/v1
    kind: DeploymentConfig
    name: image-registry
```